packages/ckeditor5-indent/src/indentblockcommand.ts

```diff
@@ -92,10 +94,17 @@ export default class IndentBlockCommand extends Command {
 /**
  * Returns blocks from selection that should have blockIndent selection set.
  */
-function getBlocksToChange( model: Model ): Array<Element> {
+function getBlocksToChange( model: Model, isForward: boolean ): Array<Element> {
```

Comment:
We could make a private method of this function so we would not need to pass so many contextual parameters.